naff/client/client.py

```diff
@@ -555,6 +555,10 @@ async def generate_prefixes(bot, message):
     def _queue_task(self, coro: Listener, event: BaseEvent, *args, **kwargs) -> asyncio.Task:
         async def _async_wrap(_coro: Listener, _event: BaseEvent, *_args, **_kwargs) -> None:
             try:
+                if not isinstance(_event, RawGatewayEvent):
```

Comment:
`Error` should also be exempt

Reply:
Resolved in 6a27c55387cc367b20ec5b526a49e2ec0b8fe4ff